From dd24ac465e99f2056eb4d17f83ae75ce6a58be1f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 02 Oct 2019 10:55:40 +0300
Subject: [PATCH] Revert "[SW-4185] gaudi: move the code to scratch pad SRAM"

We are going to remove the W/As, so return the preboot to the regular
SRAM.

This reverts commit c12d1681404ac07b57de8a9bf94b81ba244a9269.

Change-Id: I1f532011cefd7ed9bb06929f2c250597b307d02c
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/ppboot.lds.S b/gaudi/ppboot.lds.S
index 3c198df..8226163 100644
--- a/gaudi/ppboot.lds.S
+++ b/gaudi/ppboot.lds.S
@@ -5,7 +5,7 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
-MEMORY { .sram : ORIGIN = 0x7FFBFE0000, LENGTH = 0x10000 }
+MEMORY { .sram : ORIGIN = 0x7FF0000000, LENGTH = 0x10000 }
 
 OUTPUT_FORMAT("elf64-littleaarch64", "elf64-littleaarch64", "elf64-littleaarch64")
 OUTPUT_ARCH(aarch64)
diff --git a/gaudi/preboot.lds.S b/gaudi/preboot.lds.S
index 17a9257..61e3d10 100644
--- a/gaudi/preboot.lds.S
+++ b/gaudi/preboot.lds.S
@@ -5,7 +5,7 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
-MEMORY { .sram : ORIGIN = 0x7FFBFE4000, LENGTH = 0x10000 }
+MEMORY { .sram : ORIGIN = 0x7FF0004000, LENGTH = 0x10000 }
 
 OUTPUT_FORMAT("elf64-littleaarch64", "elf64-littleaarch64", "elf64-littleaarch64")
 OUTPUT_ARCH(aarch64)
